tests/conftest.py

```diff
@@ -126,3 +138,51 @@ def sample_request_with_user(user) -> Request:  # pylint: disable=redefined-oute
         is_base64_encoded=False,
         uri_params={},
     )
+
+
+@pytest.fixture()
+def sample_event_with_full_access_auth_header(
```

Comment:
I know `sample_event_with_limited_access_auth_header` is used in more places but still, replacing it with the similar line I mentioned above, will simplify the tests - you won't have to check the fixture and understand magic parameterization.

Maybe it can be named _duplication_ but it is only _apparent duplication_ - there is no possibility to make everything generic. In general, in the tests, we should follow simplicity instead of trying to write not duplicate code (especially when we talk about one line 😉 ).